tests/lib/nested.sh

```diff
@@ -679,7 +693,7 @@ nested_start_core_vm_unit() {
         if [ "$NESTED_ENABLE_SECURE_BOOT" = "true" ]; then
             cp -f "/usr/share/OVMF/OVMF_VARS.$OVMF_VARS.fd" "$NESTED_ASSETS_DIR/OVMF_VARS.$OVMF_VARS.fd"
             PARAM_BIOS="-drive file=/usr/share/OVMF/OVMF_CODE.$OVMF_CODE.fd,if=pflash,format=raw,unit=0,readonly -drive file=$NESTED_ASSETS_DIR/OVMF_VARS.$OVMF_VARS.fd,if=pflash,format=raw"
-            PARAM_MACHINE="-machine q35${ATTR_KVM} -global ICH9-LPC.disable_s3=1"
+            PARAM_MACHINE="-machine q35${ATTR_KVM},smm=on,mem-merge=off -global ICH9-LPC.disable_s3=1"
```

Comment:
This change seems unrelated to the commit message. Is this on purpose? If so I think putting it into it's own commit (with an explaination) would be better.

Reply:
sure, thanks for the review